app/apptesting/gamm.go

```diff
@@ -156,6 +182,24 @@ func (s *KeeperTestHelper) PrepareBalancerPoolWithPoolAsset(assets []balancer.Po
 	return poolId
 }
 
+// PrepareBalancerPoolWithPoolAssetAndSwapFee sets up a Balancer pool with an array of assets and a set swap fee.
+func (s *KeeperTestHelper) PrepareBalancerPoolWithPoolAssetAndSwapFee(assets []balancer.PoolAsset, swapFee sdk.Dec) uint64 {
```

Comment:
Just commenting to think about this in the future.

There are so many variations of these helpers now. We should think about reducing their number while keeping the same functionality.

Reply:
Agreed, should I make a "good first issue" to combine them into a single helper that allows modification of all pool traits?

Reply:
Yes! That would be great, thank you!